clients/bigquery/dialect/dialect.go

```diff
@@ -249,6 +249,11 @@ MERGE INTO %s %s USING %s AS %s ON %s`,
 		tableID.FullyQualifiedName(), constants.TargetAlias, subQuery, constants.StagingAlias, strings.Join(equalitySQLParts, " AND "),
 	)
 
+	cols, removed := columns.RemoveOnlySetDeleteColumnMarker(cols)
+	if !removed {
+		return []string{}, errors.New("artie only_set_delete flag doesn't exist")
+	}
```

Comment:
Bigquery isn't using this new marker yet, but still has to filter it out so it doesn't end up as a column in the destination table (same for mssql and redshift below)